metadata-protobuf/proto/WorkingGroups.proto

```diff
@@ -63,8 +63,12 @@ message ModeratePost {
   required string rationale = 2;
 }
 
+message VerifyValidator{
+  required uint64 memberId = 1;
```

Comment:
This is enough in theory but please also add the validator account too. So in the mappings you can make sure the validator account on the member metadata is actually the one the HR worker intends to verify (and it wasn't switched at some point).